.gitignore

```diff
@@ -87,4 +87,7 @@ dev/crab
 #!tutorials/images/*
 *.fits
 *.fits.gz
+*.fits_arf.gz
```

Comment:
Oops this is a bug that I introduced in #3152. It seems `pathlib` does not recognise `.fits.gz` as a suffix but just `.gz`. I'll fix this in a separate PR and the you can remove this again...

Reply:
Fixed in master: https://github.com/gammapy/gammapy/commit/1165717638b54712292d46b501e712e39f9f63eb